packages/block-editor/src/components/block-list/block-popover.js

```diff
@@ -124,10 +124,14 @@ function BlockPopover( {
 			className="block-editor-block-list__block-popover"
 			__unstableSticky={ showEmptyBlockSideInserter ? false : popoverIsSticky }
 			__unstableSlotName="block-toolbar"
+			__unstableBoundaryParent
 			// Allow subpixel positioning for the block movement animation.
 			__unstableAllowVerticalSubpixelPosition={ moverDirection !== 'horizontal' && node }
 			__unstableAllowHorizontalSubpixelPosition={ moverDirection === 'horizontal' && node }
```

Comment:
It feels like we have too many unstable props for the Popover

Reply:
Yeah 😅 Soon we should look into rewriting `Popover` or swapping it out, but I'd like to wait for some more simplifications and use cases for positioning.